am-lib/src/test/java/uk/gov/hmcts/reform/amlib/helpers/TestConstants.java

```diff
@@ -21,6 +23,7 @@
     public static final String SECURITY_CLASSIFICATION = "Public";
     public static final String ACCESSOR_ID = "a";
     public static final String OTHER_ACCESSOR_ID = "b";
+    public static final List<String> ROLE_NAME = Arrays.asList("case");
```

Comment:
A `ROLE_NAME` suggest single value not a single element list :( Please improve.

Reply:
👍

Reply:
I think should stay ROLE_NAME for now due to consistency. Currently it does return one role name. In future iterations i believe this can easily change but for now and testing purposes ROLE_NAME makes sense